examples/chat/src/main.rs

```diff
@@ -156,14 +156,15 @@ fn main() {
     }
 
     // Configure network
+    let max_message_size = 1024; // 1 KB
```

Comment:
nit: Given its usage it multiple places, I'd recommend making this a `cont` (may also want to do the same for "message backlog" -> "128 messages inflight" value)